src/runtime/storageNG/storage-proxy.ts

```diff
@@ -174,6 +174,7 @@ export class StorageProxy<T extends CRDTTypeRecord> {
           if (!this.crdt.applyOperation(op)) {
             // If we cannot cleanly apply ops, sync the whole model.
             this.clearSynchronized();
+            this.notifyUpdate(op);
```

Comment:
This case should do a first pass over all handles to notify those with `!keepSynced && notifyUpdate`, then there's no need to have this new `notifyUpdate` call.

Reply:
done